docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -16,7 +16,24 @@ See the section on [upgrading clusters][upgrades].
 
 See the section on [Adopting existing machines into KubeadmControlPlane management][adoption]
 
+### Running workloads on control plane machines
+
+The user may want to run non-control plane workloads on control plane machines.
+
+In this case, the user is also the ultimate responsible for ensuring the
```

Comment:
Maybe we can write it like "the user is ultimately responsible for ensuring ..."